openslides_backend/action/actions/user/create_update_permissions_mixin.py

```diff
@@ -190,9 +210,13 @@ def check_permissions(self, instance: Dict[str, Any]) -> None:
         if "forwarding_committee_ids" in instance:
             raise PermissionDenied("forwarding_committee_ids is not allowed.")
 
-        permstore = PermissionVarStore(self.datastore, self.user_id)
-        if permstore.user_oml == OrganizationManagementLevel.SUPERADMIN:
-            return
+        if not hasattr(self, "permstore"):
```

Comment:
`hasattr` is always kind of hacky. How about an optional constructor parameter which you can check normally and create the perm store if it is `None`?

Then you could also remove the verbose constructor in `CreateUpdatePermissionsFailingFields`.

Reply:
This is made, because in participant import I need the `permstore` for each import line, always with the same user. No need to integrate the `permstore` in all actions.

Reply:
We decided to use the `hasattr`, because the other variant expects 19 type casts by mypy